boxoffice/views/participant.py

```diff
@@ -35,4 +41,5 @@ def assign(order):
         phone=assignee_dict.get('phone'), details=assignee_details, line_item=line_item)
         db.session.add(assignee)
     db.session.commit()
+    boxofficeq.enqueue(send_ticket_assignment_email, order.id, line_item.id)
```

Comment:
Also, you can just pass `line_item.id`. You can derive `order` with `line_item.order`.